relayer/src/link/relay_path.rs

```diff
@@ -1419,6 +1433,9 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
     /// If the relaying path has non-zero packet delays, this method also updates the client on the
     /// target chain with the appropriate headers.
     fn schedule_operational_data(&self, mut od: OperationalData) -> Result<(), LinkError> {
+        let span = span!(Level::INFO, "schedule", id = %od);
```

Comment:
```suggestion
        let span = span!(Level::INFO, "schedule", id = %od.tracking_id());
```

Reply:
Same as my comment for the `relay` span. There is additional context beyond the tracking ID which we'd like to represent.

> 2021-12-22T12:18:40.069556Z DEBUG ThreadId(443) schedule{id=BS6CBKeio4 ->Destination @0-25087; len=1}: [ibc-0:transfer/channel-0 -> ibc-1] connection delay is zero: client update message will be prepended later

Shall we have KVs?

Reply:
Used `odata` in 85c60fa8f1b37e0f823aaa3cf03c79bafd8ef994.